project_ux/views/project_task_views.xml

```diff
@@ -0,0 +1,83 @@
+                </h2>
+            </h1>
+            <field name="partner_id" position="after">
+                <field name="template_task"/>
+            </field>
+        </field>
+    </record>
+
+    <record id="view_task_form2_subtask" model="ir.ui.view">
+        <field name="name">project.task2.inherit.subtask</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="view_task_form2"/>
+        <field name="priority">90</field>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <field name="parent_id" position="replace"/>
+            <field name="project_id" position="before">
+                <field name="parent_id" required="True" domain="[('project_id', '=', project_id)]"/>
+            </field>
+        </field>
+    </record>
+
+    <record id="view_task_kanban_inherit" model="ir.ui.view">
+        <field name="name">project.task.kanban.inherit</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_kanban"/>
+        <field name="priority">90</field>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <kanban position="attributes">
+                <attribute name="on_create"></attribute>
+            </kanban>
+        </field>
+    </record>
+
+    <record id="view_task_search_form" model="ir.ui.view">
+        <field name="name">project.task.search.inherit</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_search_form"/>
+        <field name="arch" type="xml">
+            <filter name="my_tasks" position="after">
+                <filter string="Is Task" name="is_task" domain="[('parent_id','=', False)]"/>
+                <filter string="Is Sub-Task" name="is_sub_task" domain="[('parent_id','!=', False)]"/>
+            </filter>
+        </field>
+    </record>
+
+    <record id="act_project_project_2_project_sub_task_all" model="ir.actions.act_window">
+        <field name="name">Sub Tasks</field>
+        <field name="res_model">project.task</field>
+        <field name="view_mode">kanban,tree,form,calendar,pivot,graph</field>
+        <field name="context">{
+            'group_by': 'stage_id',
+            'search_default_project_id': [active_id],
+            'default_project_id': active_id,
+        }</field>
+        <field name="domain">[('parent_id','!=', False)]</field>
+        <field name="view_ids"
+                   eval="[(5,0,0),(0, 0, {'view_mode': 'kanban', 'view_id': ref('view_task_kanban_inherit')}),
+                   (0, 0, {'view_mode': 'form', 'view_id': ref('view_task_form2_subtask')})]"/>
+        <field name="search_view_id" ref="project.view_task_search_form"/>
+    </record>
+
+    <record id="project.act_project_project_2_project_task_all" model="ir.actions.act_window">
```

Comment:
This action is repeated, is in both `project_project_views.xml` and `project_task_views.xml`